pcp/inventory.md

```diff
@@ -51,6 +52,9 @@ An inventory response message is decribed by the following json-schema:
 }
 ```
 
+The envelope `content` of inventory response messages must include the optional
+`in-reply-to` entry, set to the `id` of the inventory request message.
```

Comment:
It does sound correct to my foreign ears, but I'll remove "the optional" (only 16 results in Google Scholar, I surrender).